actioncable/lib/rails/generators/channel/templates/assets/channel.js

```diff
@@ -0,0 +1,21 @@
+(function() {
```

Comment:
Do we want/need this wrapper function? We're not declaring anything that can leak as a global, so I guess it comes down to whether we generate so-wrapped JS elsewhere.